crates/sp1/evm-exec/script/src/bin/main.rs

```diff
@@ -41,6 +43,12 @@ struct Args {
 
     #[arg(long)]
     height: u64,
+
+    #[arg(long)]
+    trusted_height: Option<u64>,
+
+    #[arg(long)]
+    trusted_root: Option<String>,
```

Comment:
I still need to document why I added these, but that shouldn't stop PR review. 

Essentially, a celestia block may have zero EthClientExecutorInputs for evm blocks in it. In that event we can't just take the parent headers root/height in this script so best to allow via flags.

Reply:
Done